README.md

```diff
@@ -16,24 +76,116 @@ O CLI mostra informações sobre o mundo de Game of Thrones utilizando, como fon
 
 O código deste repositório possui a funcionalidade de listar as personagens, e exibir os detalhes sobre uma personagem selecionada. Além de refatorar o código já existente, você deverá adicionar a funcionalidade de buscar livros pelo nome, exibir os resultados, e mostrar os detalhes do livro selecionado.
 
+### Data de Entrega
+
+O projeto tem até a seguinte data: `DD/MM/YYYY - 14:00h`. Para ser entregue a avaliação final.
+
 ## Requisitos do projeto
 
-### 1 - O projeto deve ser feito, necessariamente, utilizando Promises
+### 💡Veja o exemplo a seguir de como o projeto pode se parecer depois de pronto. Lembre-se que você pode ~~e deve~~ ir além para deixar o projeto com a sua cara e impressionar à todos!
 
-Não pode haver nenhum tipo de código síncrono ou que utilize callbacks.
+![exemplo do resultado](./public/funcionamento_do_projeto.gif)
+
+## ⚠️ Leia-os atentamente e siga à risca o que for pedido. Em particular. ⚠️
+
+O não cumprimento de um requisito, total ou parcialmente, impactará em sua avaliação.
+
+---
+
+### 👀Observações importantes:
+
+Para cada menu novo que você adicionar, crie um diretório dentro de `lib/menus`, e adicione os arquivos do menu lá. Por exemplo, crie um menu `books` e adicione-o ao index da pasta `menus`:
+
+*A estrutura do projeto deve ser a mesma do menu character*
```

Comment:
_Menu character_ é um projeto? Se sim, acho que vale usar o nome certinho, _Projeto Menu Character_ ou o que for. Que que vc acha?

Reply:
esse e um menu que ja vem pronto no projeto com a seguinte estrutura
lib/menus/character
lib/menus/character/index.js
lib/menus/character/actions
lib/menus/character/actions/list.js
lib/menus/character/actions/index.js

então os alunos irão seguir esse modelo